python/lsst/afw/display/interface.py

```diff
@@ -816,26 +805,27 @@ def zoom(self, zoomfac=None, colc=None, rowc=None, origin=afwImage.PARENT):
             self._impl._zoom(zoomfac)
 
     def pan(self, colc=None, rowc=None, origin=afwImage.PARENT):
-        """Pan to a location
+        """Pan to a location.
 
         Parameters
         ----------
         colc, rowc
-            the coordinates to pan to
+            Coordinates to pan to.
         origin : `lsst.afw.image.ImageOrigin`
             Coordinate system for the given positions.
 
         See also
         --------
         Display.zoom
         """
-
         self.zoom(None, colc, rowc, origin)
 
     def interact(self):
-        """Enter an interactive loop, listening for key presses in display and firing callbacks.
+        """Enter an interactive loop, listening for key presses in display and
+        firing callbacks.
 
-        Exit with ``q``, ``CR``, ``ESC``, or any other callback function that returns a `True` value.
+        Exit with ``q``, ``CR``, ``ESC``, or any other callback function that
```

Comment:
Exit with ``q``, ``CR``, ``ESC``, or any equivalent UI action provided in the display.  The loop may  also be exited by returning `True` from a user-provided callback function.